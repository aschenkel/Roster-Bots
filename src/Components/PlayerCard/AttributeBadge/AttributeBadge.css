.badge{
    background-color: royalblue;
    width: 30px;
    height: 35px;
    border-radius: 50%;
    padding: 8px;
    color: white

}
