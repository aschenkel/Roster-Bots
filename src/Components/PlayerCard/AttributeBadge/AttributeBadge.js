import React from 'react';
import PropTypes from 'prop-types';
import './AttributeBadge.css';

const AttributeBadge = (props) => {
    return (
        props.speed ?
            <p className="badge speedBadge">{props.value}</p>
        :
            <p className="badge">{props.value}</p>
    );
}

AttributeBadge.propTypes = {
  value: PropTypes.number.isRequired,
  speed: PropTypes.bool

};


export default AttributeBadge;