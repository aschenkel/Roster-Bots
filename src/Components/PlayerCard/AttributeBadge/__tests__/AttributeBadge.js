import React from 'react';
import AttributeBadge from '../AttributeBadge';
import renderer from 'react-test-renderer';

it('should render a playerÂ´s attribute (speed)', () => {
    const tree = renderer.create(
        <AttributeBadge value={44}/>
        ).toJSON();
    expect(tree).toMatchSnapshot();
});


