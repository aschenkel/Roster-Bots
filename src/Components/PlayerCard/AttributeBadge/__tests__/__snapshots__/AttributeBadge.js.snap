// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a playerÂ´s attribute (speed) 1`] = `
<p
  className="badge"
>
  44
</p>
`;
